app/core/src/navigation/index.js

```diff
@@ -30,16 +29,32 @@ const VoidStack = StackNavigator(
   },
 );
 
-const Navigation = StackNavigator(
+const tabNavigator = TabNavigator(
+  {
+    Search: { screen: VoidStack },
+    Bookings: { screen: MMBStack },
+    Message: { screen: VoidStack },
+    Profile: { screen: VoidStack },
+  },
+  {
+    tabBarOptions: {
+      activeTintColor: Color.brand,
+      inactiveTintColor: 'gray',
+    },
+    initialRouteName: 'Bookings',
+  },
+);
+
+export default StackNavigator(
   {
     Homepage: {
       screen: HomepageStack,
     },
     HotelsPackage: {
-      screen: withMappedProps(HotelsPackageWrapper),
+      screen: HotelsStack,
     },
-    SingleHotelPackage: {
-      screen: withMappedProps(SingleHotelsPackageWrapper),
+    MMBPackage: {
+      screen: tabNavigator,
```

Comment:
I don't understand this change. My initial intention was to go closer to our native app by creating the tab navigator (as we have in our app) so we can say later - _"look, this is what we have and it's basically the same with native"_. So I wanted to use even the homepage as a homepage (even though we have only links there so far). What do you think about that?

Reply:
Ait, I guess I can work with that.

Reply:
The change is simple - just keep the homepages as we do (we could actually remove some links) and let's work on the second tab (the same with our native app).

Later on, we will have to introduce the horizontal scroll view with suggested hotels and we can directly use it on our homepage...